week-1-ts-ecosystem/w01e01-typescript-basics/src/types/index.d.ts

```diff
@@ -0,0 +1,11 @@
+interface Address {
+  street: string;
+  no: number;
+}
+
+export interface User {
+  name: string;
+  age: number;
+  email: string;
+  address: Address;
```

Comment:
z utilsów wynika że adres może być nieobecny w obiekcie `user`

```suggestion
  address?: Address;
```